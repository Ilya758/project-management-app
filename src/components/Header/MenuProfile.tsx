import { NavLink } from 'react-router-dom';
import './Header.css';

export const Menu = () => {
  return (
    <div className="nav-container-link">
      <NavLink to="/board">Create new board</NavLink>
      <NavLink to="/edit_profile">Edit profile</NavLink>
      <NavLink to="/login">logout</NavLink>
      <button className="delete-user">Delete User</button>
    </div>
  );
};
