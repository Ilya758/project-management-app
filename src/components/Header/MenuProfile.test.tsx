import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import { Menu } from './MenuProfile';

describe('Menu Component', () => {
  it('Should render', () => {
    const component = render(
      <BrowserRouter>
        <Menu />
      </BrowserRouter>
    );
    expect(component.baseElement).toMatchSnapshot();
  });
});
