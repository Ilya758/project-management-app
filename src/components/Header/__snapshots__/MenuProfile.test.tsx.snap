// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Component Should render 1`] = `
<body>
  <div>
    <div
      class="nav-container-link"
    >
      <a
        class=""
        href="/board"
      >
        Create new board
      </a>
      <a
        class=""
        href="/edit_profile"
      >
        Edit profile
      </a>
      <a
        class=""
        href="/login"
      >
        logout
      </a>
      <button
        class="delete-user"
      >
        Delete User
      </button>
    </div>
  </div>
</body>
`;
